sync15-adapter/examples/sync-pass.rs

```diff
@@ -498,16 +510,21 @@ fn prompt_record_id(e: &PasswordEngine, action: &str) -> Option<String> {
 
 fn main() -> Result<(), failure::Error> {
     env_logger::init();
-    let oauth_data = do_auth(false)?;
 
-    let scope = &oauth_data.keys["https://identity.mozilla.com/apps/oldsync"];
+    let cfg = Config::release()?;
+    let tokenserver_base_url = cfg.token_server_endpoint_url()?.join("../../")?.as_str().to_string();
+
+    let mut acct = do_auth(cfg)?;
+    let token = acct.get_oauth_token(&[SYNC_SCOPE])?.unwrap();
+    let keys: HashMap<String, ScopedKeyData> = serde_json::from_str(&token.keys.unwrap())?;
```

Comment:
Having to deserialize this from a string was kind of weird, but I guess this struct is more designed to go cleanly across an FFI boundary than to be consumed directly by rust code?

Reply:
Yup, we could make the change if you feel strongly about it though